bot/tests/tasks/test_pennychat.py

```diff
@@ -194,105 +196,84 @@ def __call__(self, channel, blocks):
-        slack_id='organizer',
-        display_name='organizer',
-        user=organizer_user,
-        slack_team_id=SLACK_TEAM_ID,
-    )
-    SocialProfile.objects.create(
-        slack_id='invitee_1',
-        display_name='invitee_1',
-        user=invitee_1_user,
-        slack_team_id=SLACK_TEAM_ID,
-    )
-    SocialProfile.objects.create(
-        slack_id='invitee_2',
-        display_name='invitee_2',
-        user=invitee_2_user,
-        slack_team_id=SLACK_TEAM_ID,
-    )
-    imminent_chat.save_participant(organizer_user, Participant.ORGANIZER)
-    imminent_chat.save_participant(invitee_1_user, Participant.ATTENDEE)
-    imminent_chat.save_participant(invitee_2_user, Participant.ATTENDEE)
-
-    slack_client = mocker.Mock()
-
-    with mocker.patch('bot.tasks.pennychat._penny_chat_details_blocks', return_value='<blocks>'), \
-            mocker.patch('bot.tasks.pennychat.get_slack_client', return_value=slack_client):
-        send_penny_chat_reminders()
-
-    actual = {call[1]['channel'] for call in slack_client.chat_postMessage.call_args_list}
-    expected = {'organizer', 'invitee_1', 'invitee_2'}
-    assert expected == actual
-
-
-@pytest.mark.django_db
-def test_send_penny_chat_reminders__send_NO_message_to_addendees_of_past_events(mocker):
-    imminent_chat = PennyChatSlackInvitation.objects.create(
-        title='Chat 1',
-        description='The first test chat',
-        date=datetime.now().astimezone(TIMEZONE) - timedelta(minutes=10),
-        status=PennyChatSlackInvitation.SHARED,
-        organizer_slack_id='organizer',
-        created_from_slack_team_id=SLACK_TEAM_ID,
-    )
-    organizer_user = User.objects.create_user('organizer')
-    invitee_1_user = User.objects.create_user('invitee_1')
-    invitee_2_user = User.objects.create_user('invitee_2')
-    SocialProfile.objects.create(
-        slack_id='organizer',
-        display_name='organizer',
-        user=organizer_user,
-        slack_team_id=SLACK_TEAM_ID,
-    )
-    SocialProfile.objects.create(
-        slack_id='invitee_1',
-        display_name='invitee_1',
-        user=invitee_1_user,
-        slack_team_id=SLACK_TEAM_ID,
-    )
-    SocialProfile.objects.create(
-        slack_id='invitee_2',
-        display_name='invitee_2',
-        user=invitee_2_user,
-        slack_team_id=SLACK_TEAM_ID,
-    )
-    imminent_chat.save_participant(organizer_user, Participant.ORGANIZER)
-    imminent_chat.save_participant(invitee_1_user, Participant.ATTENDEE)
-    imminent_chat.save_participant(invitee_2_user, Participant.ATTENDEE)
-
-    slack_client = mocker.Mock()
-
-    with mocker.patch('bot.tasks.pennychat._penny_chat_details_blocks', return_value='<blocks>'), \
-            mocker.patch('bot.tasks.pennychat.get_slack_client', return_value=slack_client):
-        send_penny_chat_reminders()
-
-    actual = {call[1]['channel'] for call in slack_client.chat_postMessage.call_args_list}
-    expected = set()
-    assert expected == actual
+testdata = [
```

Comment:
I turned 4 very redundant tests into a single parameterized test for 8 different cases.